File: yukicoder/504_ゲーム大会(ランキング)/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        int rank = 1;

        int scoreK = sc.nextInt();

        System.out.println(rank);

        for (int i = 1; i < n; i++) {

            int scoreOthers = sc.nextInt();

            if (scoreOthers > scoreK) {
                rank++;
            }

            System.out.println(rank);

        }

    }

}

File: yukicoder/504_ゲーム大会(ランキング)/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int n, scoreK;
    cin >> n >> scoreK;

    int rank = 1;

    cout << rank << endl;

    for (int i = 1; i < n; i++) {

        int scoreOthers;
        cin >> scoreOthers;

        if (scoreOthers > scoreK) {
            rank++;
        }

        cout << rank << endl;

    }

    return 0;

}
